.github/workflows/ci-example.yml

```diff
@@ -0,0 +1,41 @@
+name: CI_Example
+
+on:
+  pull_request:
+    branches:
+      ci-add-examples-test-211
+
+jobs:
+  examples-test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest] #[macos-latest, ubuntu-latest]
+        python-version: [3.7,]
+
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: true
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install .[match-py-ver] --no-cache-dir
```

Comment:
I do not think we need these dependencies

Reply:
I took these from the ci.yml and I think upgrading pip is a good idea. What is the last line doing?

Reply:
I think is installing `jina` with `py37` or `py38` dependencies, I think this will be done later to install from master. So it is one or the other in my opinion